### PR TITLE
Update readme for SMS checker: Alex

### LINKED ISSUE #119: Update SMS checker README
As a developer, I want clear documentation that outlines the installation process, the structure of the pipeline, and how to run the app, so new contributors can easily get started and understand the functionality of the app.

- Edit SMS checker README so that it is up to date with project

### PR DESCRIPTION
**Pull request by AlexOBarnes**
This closes #119 

What I have done:
- Updated README for SMS checker
- Ensured the documentation is thorough and consistent

### CODE REVIEW

**Review by MurasatoNaoya — APPROVED** [review]
Clear changes to the SNS functionality's README. Formatting is appropriate and understandable; warrants no further comment. LGTM.

File: sms-checker/README.md [hunk 1]
```diff
@@ -2,7 +2,86 @@
...
+```
+3. Create a `.env` file with the following:
+    - DB_HOST - to access your RDS instance
+    - DB_USER - username for accessing RDS
+    - DB_PASSWORD - password for accessing RDS
+    - DB_NAME - the name of your database
+    - DB_PORT - port to access RDS (typically 5432 for postgres)
+    - AWS_ACCESS_KEY - your personal access key to AWS
+    - AWS_SECRET_KEY - your personal secret key to AWS
+    - REGION - the region of AWS that you will be using for this project
```
Comment (CalebMathers) [inline sms-checker/README.md:23]: Very informative, good to tell people exactly what the env variables require despite the descriptive names

File: sms-checker/checker.py [hunk 2]
```diff
@@ -23,7 +23,7 @@ def get_subscribers() ->list[dict]:
     JOIN body as b USING (body_id)
     WHERE f.visibility_m > 500 AND
     f.cloud_coverage_percent < 25 AND
-    ba.at <= CURRENT_TIMESTAMP + INTERVAL '4 hours'
+    ba.at <= CURRENT_TIMESTAMP + INTERVAL '3 hours'
```
Comment (CalebMathers) [inline sms-checker/checker.py:26]: Good interval

File: sms-checker/message.py [hunk 3]
```diff
@@ -60,7 +60,7 @@ def get_client(service: str) -> client:
     '''Returns a AWS client'''
     return client(service, aws_access_key_id=ENV["MY_AWS_ACCESS_KEY"],
                         aws_secret_access_key=ENV["MY_AWS_SECRET_KEY"],
-                        region_name=ENV['AWS_REGION'])
+                        region_name=ENV['REGION'])
```
Comment (CalebMathers) [inline sms-checker/message.py:63]: Take that Jeff, you don't own the regions!

File: sms-checker/README.md [hunk 4]
```diff
@@ -2,7 +2,86 @@
...
+
+5. Add to your `.env` file the following:
+    - ECR_URI - endpoint of the ECR repository
+    - IMAGE_NAME - a name given to your docker image
+
+6. Run the following command to automatically dockerise and push to ECR:
+```bash
+bash ../dashboard/deploy.sh
+```
+7. Now follow the terraform setup steps in the `../terraform` folder to provision a lambda based off this docker image
```
Comment (CalebMathers) [inline sms-checker/README.md:49]: Could change this to link to the terraform directory

**Review by CalebMathers — APPROVED** [review]

**Pull request merged**